content/docs/dev/tasks/configure/security.md

```diff
@@ -101,6 +101,9 @@ ca-path = "/path/to/ca.pem"
 cert-path = "/path/to/tikv-server-cert.pem"
 # The path to the file that contains the PEM encoding of the server’s private key.
 key-path = "/path/to/tikv-server-key.pem"
+# The name list used to verify the common name in client’s certificates. Verification is
+# not enabled if this field is empty.
```

Comment:
Can you give some examples of the names?

Reply:
Done